generated/nidaqmx/constants.py

```diff
@@ -121,8 +121,8 @@ class BridgePhysicalUnits(Enum):
     POUNDS = 15876  #: Pounds.
     KILOGRAM_FORCE = 15877  #: kilograms-force.
     PASCALS = 10081  #: Pascals.
+    POUNDS_PER_SQUARE_INCH = 15879  #: Pounds per square inch.
```

Comment:
I like this change... shorthand SQ became SQUARE. I think that is a good and more consistent change.

Reply:
`BridgePhysicalUnits.POUNDS_PER_SQ_INCH` is used in nidaqm-python repo. Thus have retained the expected enum value name (`POUNDS_PER_SQ_INCH`)